### ISSUE TITLE
Poplife

### ISSUE THREAD

**Issue by Superfly1337** [Akryllax/project-purelife#1]
Do you know how we can get i contact with anyone that have the source file for metropolis v2 lakeside map?

**Comment by Akryllax** [comment]
Hello, you mean the PopLife V2.0 map we used back in the day in Plata o Plomo community?

I detached from the project early but I'll go check if I have some backups available, and I'll ask around. Give me one day or two :)

**Comment by Superfly1337** [comment]
Yes, thats the one i been looking for 👍i appreciate all your help
Here: https://www.youtube.com/watch?v=v-mbPWWvXRc

**Comment by Superfly1337** [comment]
any updates?

**Comment by Superfly1337** [comment]
?

**Comment by Akryllax** [comment]
Hello! Sorry I completely missed this. Tried locating a few of the former
devs to catch up but no luck, I am sorry! Also looked into backups that I
had, no trace of the project. Good luck!
